packages/core/src/events/TapEvent.ts

```diff
@@ -333,7 +322,9 @@ const applyEvent = (target: Global): void => {
    * Use this to fire tap events, if `enter` or `space` was pressed on button
    */
   target.addEventListener('click', (event: MouseEvent | PointerEvent) => {
-    if (isButtonEnterOrSpace(event)) {
+    // check for click, dbclick, mousedown, mouseup events
+    // as described in https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/detail
+    if (event.detail === 0) {
```

Comment:
The code comment is not correct, it must be "check for events from enter or space button"

Reply:
comment updated.